doc/coq.rst

```diff
@@ -28,12 +28,14 @@ collection of Coq theories and plugins.
 The ``.v`` files of a theory need not be present as source files. They may also
 be Dune targets of other rules.
 
+Coq theories may come from the same workspace or from an install location.
```

Comment:
Maybe "local" workspace? I would expand here a bit more as to illustrate what we mean, for example

"Dune will locate Coq theories first by looking at the set of stanzas defined in the current workspace plus the set of theories globally available for a given Coq install, similarly to what is done for OCaml.

Theories in the same scope will be used first, then public theories in the same workspace will be referenced, and if the desired theory is not found there, Dune will scan the `COQPATH` and `user-contrib` Coq global location to try to find the required theory"